*********************************************************************************************************************************************
04.05.2016 19:43:06 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 73
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 64
   в DynamicInjectord743f22b31d44b4fbce8381df2ba8c98(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 19:44:03 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 73
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 64
   в DynamicInjector7c5f795c3dd243b288a3d1058ed0ae5f(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 19:47:31 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 72
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 64
   в DynamicInjector9263f54d9d494c7a8a5b2a735a69dccf(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 19:48:57 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 73
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 65
   в DynamicInjectora789c93a11374ad2a7ffb505a3eb20bf(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 20:04:17 DESKTOP-KRUHAN2 darks
Произошла одна или несколько ошибок.
mscorlib
   в System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions)
   в System.Threading.Tasks.Task.Wait(Int32 millisecondsTimeout, CancellationToken cancellationToken)
   в System.Threading.Tasks.Task.Wait()
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 82
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 65
   в DynamicInjector83622d3b592f4e31b7450511589caf6e(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 20:17:39 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 74
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 65
   в DynamicInjector14cffb9479de419786aa71b703e2ee4d(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 20:21:30 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 77
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 65
   в DynamicInjectord5d35d428ed84774a01bb74433ee61a4(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 20:33:19 DESKTOP-KRUHAN2 darks
Значение не может быть неопределенным.
Имя параметра: source
System.Core
   в System.Linq.Enumerable.Count[TSource](IEnumerable`1 source)
   в iim.Presenter.GetStores() в c:\dev\iim\iim\Presenter\Presenter.cs:строка 73
   в iim.Presenter..ctor(ICore core) в c:\dev\iim\iim\Presenter\Presenter.cs:строка 65
   в DynamicInjectorf2e2ddd9664440dab147ce14bf2f883a(Object[] )
   в Ninject.Activation.Providers.StandardProvider.Create(IContext context)
   в Ninject.Activation.Context.Resolve()
   в Ninject.KernelBase.<>c__DisplayClass10.<Resolve>b__c(IBinding binding)
   в System.Linq.Enumerable.WhereSelectEnumerableIterator`2.MoveNext()
   в System.Linq.Enumerable.<CastIterator>d__94`1.MoveNext()
   в System.Linq.Enumerable.Single[TSource](IEnumerable`1 source)
   в Ninject.ResolutionExtensions.Get[T](IResolutionRoot root, IParameter[] parameters)
   в iim.Program.Main() в c:\dev\iim\iim\Program.cs:строка 34
*********************************************************************************************************************************************
04.05.2016 20:35:33 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:35:48 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:36:13 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:38:43 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:38:59 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:40:00 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:40:15 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:40:40 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:40:55 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:49:06 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 20:49:22 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:22:14 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:22:29 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:25:51 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:26:06 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:30:27 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:38:50 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:39:06 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:39:49 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:40:04 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:46:29 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 21:46:44 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:03:56 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:04:12 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:05:33 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:05:49 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:07:54 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:08:09 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:10:02 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:10:17 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:11:25 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в d:\Projects\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:11:41 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в d:\Projects\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:53:36 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в c:\dev\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в c:\dev\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:53:52 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в c:\dev\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:59:35 DESKTOP-KRUHAN2 darks
Прослушивание на http://192.168.0.13:666/test/GetStoresList не выполняла ни одна конечная точка, которая могла бы принять сообщение. Среди прочих причин это могло быть вызвано неправильным адресом или действием SOAP. Подробнее см. в описании InnerException (если имеется).
mscorlib

Server stack trace: 
   в System.ServiceModel.Channels.HttpChannelUtilities.ProcessGetResponseWebException(WebException webException, HttpWebRequest request, HttpAbortReason abortReason)
   в System.ServiceModel.Channels.HttpChannelFactory`1.HttpRequestChannel.HttpChannelRequest.WaitForReply(TimeSpan timeout)
   в System.ServiceModel.Channels.RequestChannel.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Dispatcher.RequestChannelBinder.Request(Message message, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs, TimeSpan timeout)
   в System.ServiceModel.Channels.ServiceChannel.Call(String action, Boolean oneway, ProxyOperationRuntime operation, Object[] ins, Object[] outs)
   в System.ServiceModel.Channels.ServiceChannelProxy.InvokeService(IMethodCallMessage methodCall, ProxyOperationRuntime operation)
   в System.ServiceModel.Channels.ServiceChannelProxy.Invoke(IMessage message)

Exception rethrown at [0]: 
   в System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   в System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   в WcfContract.IService.GetStoresList()
   в WcfClientLibrary.WcfClient.<GetStoresList>b__0() в c:\dev\iim\WcfClientLibrary\WcfClient.cs:строка 37
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в c:\dev\iim\Trap\ExceptionTrap.cs:строка 35
*********************************************************************************************************************************************
04.05.2016 22:59:51 DESKTOP-KRUHAN2 darks
При установлении соединения с SQL Server произошла ошибка, связанная с сетью или с определенным экземпляром. Сервер не найден или недоступен. Убедитесь, что имя экземпляра указано правильно и что на SQL Server разрешены удаленные соединения. (provider: Named Pipes Provider, error: 40 - Не удалось открыть подключение к SQL Server)
.Net SqlClient Data Provider
   в System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   в System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   в System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Linq.SqlClient.SqlConnectionManager.UseConnection(IConnectionUser user)
   в System.Data.Linq.SqlClient.SqlProvider.get_IsSqlCe()
   в System.Data.Linq.SqlClient.SqlProvider.InitializeProviderMode()
   в System.Data.Linq.SqlClient.SqlProvider.System.Data.Linq.Provider.IProvider.Execute(Expression query)
   в DataAccess.SomeLinqDataContext.mpzGetMxList() в c:\dev\iim\DataAccess\SomeLinq.designer.cs:строка 90
   в DataAccess.DataProvider.<GetStoresList>b__1() в c:\dev\iim\DataAccess\DataProvider.cs:строка 51
   в Trap.ExceptionTrap.Catch[T](Func`1 func) в c:\dev\iim\Trap\ExceptionTrap.cs:строка 35
